test/unit/events.spec.ts

```diff
@@ -135,6 +137,65 @@ describe("Events", () => {
     });
   });
 
+  describe("events can be stopped almost immediately", () => {
+    let timer: SinonFakeTimers;
+    beforeEach(() => {
+      timer = sinon.useFakeTimers();
+      flickingInfo = createFlicking(horizontal.full);
+    });
+    afterEach(() => {
+      timer.restore();
+    })
+
+    it("holdStart can be stopped almost immediately", () => {
+      // Given
+      let holdCount = 0;
+      const flicking = flickingInfo.instance;
+      flicking.on(EVENTS.HOLD_START, e => {
+        expect(flicking.isPlaying()).to.be.false;
```

Comment:
@WoodNeck how about caching the result and Check value after `// Then`?

Reply:
Okay, that would be more reasonable.